src/Controller/QuizInstanceController.php

```diff
@@ -114,12 +115,15 @@ public function showOverview(Request $request, array $requestAttributes): Respon
         $quizInstanceId = $this->session->get(self::QUIZ_INSTANCE_ID);
         $quizInstance = $this->quizInstanceService->findQuiz($quizInstanceId);
         $answeredQuestions = $this->questionInstanceService->getAnsweredQuestions($quizInstanceId);
+        $isLastQuestion = count($answeredQuestions);
```

Comment:
this is a very weird naming. `is` mainly means this is a `bool`, I highly doubt this is your intention